src/Sylius/Component/Report/Renderer/RendererInterface.php

```diff
@@ -18,9 +18,9 @@
 {
     /**
      * @param ReportInterface $report
-     * @param Data $data
+     * @param Data            $data
      *
-     * @return string
+     * @return mixed
```

Comment:
In this case, if you return rendered report as string (further Response content), it should be a string only.